npm/cypress-schematic/README.md

```diff
@@ -181,7 +181,7 @@ In order to bypass the prompt asking for your e2e spec name, simply add a `--nam
 ng generate @cypress/schematic:e2e --name=login
 ```
 
-This will create a new spec file named `login.spec.ts` in the default Cypress folder location.
+This will create a new spec file named `login.cy.ts` in the default Cypress folder location.
```

Comment:
AFAIK the change from `.spec` to `.cy` was for CT not E2E. I could be wrong on this assumption but the Writing Your First E2E Test Guide in the docs uses `.spec` as well. We should probably be cohesive in whatever we choose

Reply:
No, the change to `.cy.{js,jsx,ts,tsx}` is for both ct and e2e. See https://www.notion.so/cypressdx/Changes-for-10-0-0-1dca213837b84d6ea297c2d12c615f3a#7434ee2fe3e24e769e5b2a579cc1b27f

Reply:
Easy way to remember is anything that runs in **cy**press uses **cy**.